corehq/apps/data_interfaces/forms.py

```diff
@@ -191,66 +205,89 @@ def __init__(self, *args, **kwargs):
         # when they are displayed they are required.
         self.fields['update_property_name'].label = _("Property") + '<span class="asteriskField">*</span>'
         self.fields['update_property_value'].label = _("Value") + '<span class="asteriskField">*</span>'
-
-        if AUTO_CASE_UPDATE_ENHANCEMENTS.enabled(self.domain):
-            self.allow_updates_without_closing()
-
-        self.set_case_type_choices(self.initial.get('case_type'))
         self.helper = FormHelper()
         self.helper.form_class = 'form form-horizontal'
         self.helper.label_class = 'col-sm-3 col-md-2'
         self.helper.field_class = 'col-sm-4 col-md-3'
         self.helper.form_method = 'POST'
         self.helper.form_action = '#'
-        self.helper.layout = Layout(
-            Fieldset(
-                _("Basic Information"),
-                Field(
-                    'name',
-                    ng_model='name',
-                ),
-                Field(
-                    'case_type',
-                    ng_model='case_type',
-                ),
-                hqcrispy.B3MultiField(
-                    _("Close Case") + '<span class="asteriskField">*</span>',
-                    Div(
-                        hqcrispy.MultiInlineField(
-                            'server_modified_boundary',
-                            ng_model='server_modified_boundary',
-                        ),
-                        css_class='col-sm-6',
-                    ),
-                    Div(
-                        HTML('<label class="control-label">%s</label>' %
-                             _('days after the case was last modified.')),
-                        css_class='col-sm-6',
+
+        _update_property_fields = [
+            Field(
+                'update_property_name',
+                ng_model='update_property_name',
+                css_class='case-property-typeahead',
+            ),
+            Field(
+                'property_value_type',
+                ng_model='property_value_type',
+            ),
+            Field(
+                'update_property_value',
+                ng_model='update_property_value',
+            )
+        ]
+
+        if self.enhancements_enabled:
+            self.allow_updates_without_closing()
+        else:
+            # do not allow them to edit property_value_type
+            _update_property_fields.pop(1)
```

Comment:
i think it's better to conditionally insert rather than conditionally remove since if the order changes and we forget to update the index we end up removing the wrong element with this approach (instead of just inserting in the wrong place which isn't as bad)